packages/zowe-explorer/src/uss/USSTree.ts

```diff
@@ -964,25 +965,18 @@ export class USSTree extends ZoweTreeProvider implements IZoweTree<IZoweUSSTreeN
         }
     }
 
-    public async openWithEncoding(node: IZoweUSSTreeNode): Promise<void> {
-        const ussApi = ZoweExplorerApiRegister.getUssApi(node.getProfile());
-        let taggedEncoding: string;
-        if (ussApi.getTag != null) {
-            taggedEncoding = await ussApi.getTag(node.fullPath);
-        }
-        const encoding = await promptForEncoding(node, taggedEncoding !== "untagged" ? taggedEncoding : undefined);
-
-        const docResult = await confirmForUnsavedDoc(node);
```

Comment:
The logic on this line and below related to `confirmForUnsavedDoc` should be kept.